public/client.js

```diff
@@ -38,33 +38,31 @@ function gameSetup() {
   };
 }
 
+/**
+ * Chrome Android browser does not like it when
+ * 
+ * background image positions are set to 100%. capping this to 99.6%, which seems ok
+ * 
+ * @param {number} idx
+ * 
+ * @return {number}
+ */
+function backgroundPosition(idx) {
+  return (idx * 99.6) / 3;
+}
+
 // Position the image in the right place on each tile to reassemble it on the grid, and enable click events on the tiles
 function tileSetup() {
   const tileArray = document.querySelectorAll(".tile");
-  const gameArea = document.getElementById("game-area");
-
-  tileArray.forEach(function(tile, index) {
-    // inexplicably, Chrome Android browser does not like it when some background image positions are set to 100%.
-    // therefore capping this to 99.6%, which seems to display ok
-    const backgroundPositionX = ((index % 4) * 99.6) / 3;
-    const backgroundPositionY = (Math.floor(index / 4) * 99.6) / 3;
-
+  tileArray.forEach((tile, index) => {
     tile.id = `tile-${index}`;
-    tile.style.backgroundPosition = `${backgroundPositionX}% ${backgroundPositionY}%`;
+    tile.style.backgroundPosition = `${backgroundPosition((index % 4))}% ${backgroundPosition(Math.floor(index / 4))}%`;
 
     tileState["tileLoc"][tile.id] = index;
     gameWonState["tileLoc"][tile.id] = index;
 
-    const tileNumber = index + 1;
-    tile.querySelector(".number").innerText = tileNumber;
-
-    // tile.addEventListener("click", function(e) {
```

Comment:
killed commented out code

Reply:
removing commented out code is cool - I sometimes create extra variables just to make things a bit clearer so I'm not so wild about changes like this. I get that it's not super performant, but I don't know if this truly needs to be.